app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class OrganizationsController < ApplicationController
+  skip_before_action :set_organization
+  layout 'minimal'
+
+  def index
+    # TODO: change this when having a organization habtm user relation
+    redirect_to organization_dashboard_path(current_user.organization) if current_user.organization.present?
+
+    @organizations = Organization.all
+  end
+
+  def set_organization
+    organization = Organization.find(params[:organization_id])
+    redirect_to organization_dashboard_path(organization)
+  end
```

Comment:
```suggestion
 ```

I think this was copied from the PR that I suggested styling the organizations index page. I was playing around with something and it didn't reallly work as I expected. This method is never called and should be removed.